mailpoet/lib/AdminPages/Pages/AutomationEditor.php

```diff
@@ -46,6 +47,8 @@ public function __construct(
   public function render() {
     $id = isset($_GET['id']) ? (int)$_GET['id'] : null;
 
+    $this->wp->doAction(Hooks::EDITOR_BEFORE_LOAD, (int)$id);
```

Comment:
> Are there cases where we would actually want this to fire before we confirm that a workflow exists

At the moment, I can't think of any.

> we would still need to load it from the database again in case any changes were made

That was my thinking too.